### PR TITLE
Libreoffice plumbing

### PR DESCRIPTION
**Pull request by a17r**
@akhuettel

### CODE REVIEW

File: app-text/libwps/libwps-0.4.4.ebuild [hunk 1]
```diff
@@ -28,10 +27,11 @@ DEPEND="${RDEPEND}
 src_configure() {
 	econf \
 		--disable-werror \
-		$(use_enable static-libs static) \
 		--docdir=/usr/share/doc/${PF} \
```
Comment (SoapGentoo) [inline app-text/libwps/libwps-0.4.4.ebuild]: you sure you still need `--docdir` in `EAPI=6`?

File: app-text/libwps/libwps-0.4.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		--disable-werror \
+		--with-sharedptr=c++11 \
+		$(use_enable debug) \
+		$(use_with doc docs) \
+		$(use_enable static-libs static)
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```
Comment (SoapGentoo) [inline app-text/libwps/libwps-0.4.4.ebuild]: `prune_libtool_files` is to be shunned. Use the normal find as described in: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

Then you can also remove `eutils`

File: app-text/liblangtag/liblangtag-0.6.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		$(use_enable debug) \
+		$(use_enable doc gtk-doc) \
+		$(use_enable introspection) \
+		$(use_enable static-libs static) \
+		$(use_enable test)
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```
Comment (SoapGentoo) [inline app-text/liblangtag/liblangtag-0.6.2.ebuild]: same here

File: app-office/unoconv/unoconv-99999999.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	${PYTHON_DEPS}
+	!app-text/odt2txt
+	virtual/ooo
+"
+
+DOCS=( ChangeLog CHANGELOG.md README.adoc )
+
+src_prepare() {
+	eapply "${FILESDIR}/timeout.patch"
+	eapply_user
```
Comment (SoapGentoo) [inline app-office/unoconv/unoconv-99999999.ebuild]: `default` + `PATCHES` instead

File: app-office/unoconv/unoconv-99999999.ebuild [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+	python_fix_shebang .
```
Comment (SoapGentoo) [inline app-office/unoconv/unoconv-99999999.ebuild]: ... you're fixing python shebangs in `src_prepare`? This stuff usually happens in `src_install`

File: app-text/libstaroffice/libstaroffice-0.0.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+	econf \
+		$(use_enable debug) \
+		$(use_with doc docs) \
+		$(use_enable tools) \
+		$(use_enable zlib zip)
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```
Comment (SoapGentoo) [inline app-text/libstaroffice/libstaroffice-0.0.2.ebuild]: same here

File: media-libs/libzmf/libzmf-0.0.1.ebuild [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+	econf \
+		$(use_enable debug) \
+		$(use_with doc docs) \
+		$(use_enable test tests) \
+		$(use_enable tools)
+}
+
+src_install() {
+	default
+	prune_libtool_files --all
```
Comment (SoapGentoo) [inline media-libs/libzmf/libzmf-0.0.1.ebuild]: here too

**Comment by a17r** [comment]
Added 5.2.3.2 bump after L10N tarballs have appeared. Shouldn't take long for proper release.

**Comment by akhuettel** [comment]
All pushed

**Pull request closed without merging**